docs/commands/eject.md

```diff
@@ -0,0 +1,92 @@
+# eject
+
+## Introduction
+
+The `eject` command allows you to customize components by creating a local copy of their source code in your Redocly project.
+Use this feature when you need to modify a component's styles, structure, or behavior beyond what's possible through configuration.
+
+{% partial file="../_snippets/new-product-alert-commands.md" %}
```

Comment:
Avoid markdoc-specific syntax in the CLI docs if you can, it's an open source project so many users will read the docs on GitHub or locally, we can't assume they're using the marketing website version.

Reply:
Removed but bookmarked to discuss separately